contentcuration/contentcuration/frontend/channelEdit/views/files/supplementaryLists/SupplementaryList.vue

```diff
@@ -149,7 +149,7 @@
 
   .languageTile:hover {
     /* stylelint-disable-next-line custom-property-pattern */
-    background-color: var(--v-greyBackground-base);
+    background-color:rgba(0, 0, 0, 0.04);
```

Comment:
Instead of this, could you please use a KDS color from the grey palette that matches closely to the subtlety of your change: https://design-system.learningequality.org/colors/#palette

Furthermore, in order to accomplish the above, you'll probably need to follow the 'Computed Class' example under [Usages on the same linked page](https://design-system.learningequality.org/colors/#usage), which would result in removing this CSS here.